src/main/java/com/venafi/vcert/sdk/endpoint/Authentication.java

```diff
@@ -48,7 +48,7 @@ public Authentication(String user, String password, String accessToken, String r
 	@Override
 	public String toString() {
 		return Authentication.class.getSimpleName() + "(user=" + user + ", apiKey=" + apiKey
-				+ ", password=" + (!password.isEmpty() ? "****" : "not set") + ")";
+				+ ", password=" + (password!=null && !password.isEmpty() ? "****" : "not set") + ")";
 	}
 
 }
```

Comment:
No new line at end of file.

Reply:
In fact that doesn't affect the functionallity of the class